AirCasting/Settings/ShareLogsViewModel.swift

```diff
@@ -11,7 +11,7 @@ protocol LogfileProvider {
 class ShareLogsViewModel: ObservableObject {
     @Published var shareSheetPresented: Bool = false
     var file: URL? { logFileProvider.logFileURLForSharing() }
-    @Injected private var logFileProvider: LogfileProvider
+    private var logFileProvider: LogfileProvider { LoggerBuilder.shared.store }
```

Comment:
It would be awesome if those changes related to Logger could to be moved to another PR. Do you think you could do this? I think it would be very beneficial for keeping track of changes made in the project when looking at the develop branch (where each PR is squashed to one commit).